packages/core/addon/components/navi-visualizations/line-chart.js

```diff
@@ -199,19 +200,38 @@ export default Component.extend({
   }),
 
   /**
-   * @property {Object} data - configuration for chart x and y values
+   * @property {Array} seriesData - chart series data
    */
-  dataConfig: computed('firstModel', 'seriesConfig', 'c3ChartType', function() {
+  seriesData: computed('firstModel', 'seriesConfig', function() {
     const request = get(this, 'firstModel.request');
     const rows = get(this, 'firstModel.response.rows');
     const builder = get(this, 'builder');
     const seriesConfig = get(this, 'seriesConfig.config');
-    const seriesData = builder.buildData(rows, seriesConfig, request);
+    return builder.buildData(rows, seriesConfig, request);
+  }),
+
+  /**
+   * @property {Array} seriesDataGroups - chart series groups for stacking
+   */
+  seriesDataGroups: computed('options', 'seriesData', function() {
+    const { seriesData } = this,
+      options = merge({}, DEFAULT_OPTIONS, this.options),
+      { stacked } = options.style;
+
+    return stacked && seriesData.length ? [Object.keys(seriesData[0]).filter(key => key !== 'x')] : [];
+  }),
+
+  /**
+   * @property {Object} dataConfig - configuration for chart x and y values
+   */
+  dataConfig: computed('c3ChartType', 'seriesData', 'seriesDataGroups', function() {
+    const { c3ChartType, seriesData, seriesDataGroups } = this;
```

Comment:
Can we test to make sure this doesn't break apps on Ember 2.x? I think there's been an issue in the past with this syntax when accessing computed properties

Reply:
Fixed